src/server/hook_func.c

```diff
@@ -6471,22 +6471,25 @@ add_pending_mom_allhooks_action(void *minfo, unsigned int action)
 void
 next_sync_mom_hookfiles(void)
 {
-	int	timeout_sec;
+	long	timeout_sec;
```

Comment:
Using long is fine since that's how the PBS datatype is defined. Using unsigned long or time_t might also be good options.